Makefile

```diff
@@ -32,3 +32,14 @@ test:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
 test-verbose:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race -v ./...
+
+docker-images: docks/build/Dockerfile docks/base/Dockerfile
+	docker build --rm -f ./docks/base/Dockerfile  --tag=goclang/base .
+	docker build --rm -f ./docks/build/Dockerfile --tag=goclang/build .
+
+test-all:
```

Comment:
Since this is a docker command I am wondering if this should get the "docker-" prefix as well?